milatools/cli/code_command.py

```diff
@@ -0,0 +1,22 @@
+import json
+from pathlib import Path
+from socket import timeout
```

Comment:
Doesn't appear to be used.

Reply:
Yeah sorry this was still a bit of a WIP.

Reply:
Removed this for now in fc61b941fdd5b5ed907b1f667c24a5dd1cd20c10